tests/utils/utils.py

```diff
@@ -0,0 +1,91 @@
+import json
+import logging
+from pathlib import Path
+from time import time
+from typing import Union
+
+import pandas as pd
+from starkware.starknet.testing.starknet import StarknetContract
+
+logging.basicConfig(format="%(levelname)-8s %(message)s")
+logger = logging.getLogger("timer")
+
+
+_time_report = []
+_resources_report = []
+
+
+def timeit(fun):
+    async def timed_fun(*args, **kwargs):
+        start = time()
```

Comment:
i'm not a pythonist but could we use `time.perf_counter()` here instead of time.time()? I heard it yields better results 

```python
start = perf_counter()
# logic
stop = perf_counter()
```